R/extendr-wrappers.R

```diff
@@ -73,6 +73,8 @@ test_robj_to_expr <- function(robj) .Call(wrap__test_robj_to_expr, robj)
 
 test_wrong_call_pl_lit <- function(robj) .Call(wrap__test_wrong_call_pl_lit, robj)
 
+polars_features <- function() .Call(wrap__polars_features)
```

Comment:
This is a bit a duplicate of `polars::pl$polars_info()`, is it possible to merge the two?
```r
polars::pl$polars_info()
#> r-polars package version : 0.8.0.9000
#> rust-polars crate version: 0.32.0
#> 
#> Features:          
#> simd FALSE

polars_features()
#> $full_features
#> [1] FALSE
#> 
#> $default
#> [1] TRUE
#> 
#> $simd
#> [1] FALSE
#> 
#> $rpolars_debug_print
#> [1] FALSE
```